codespell_lib/data/dictionary.txt

```diff
@@ -6865,18 +6868,23 @@ lazyness->laziness
 leaast->least
 leace->leave
 leaded->led, lead, leaded,
+leafs->leaves
```

Comment:
`leafs` is a valid word i.e. `leafs through`. I'd add it as a result as well since it's non-typical

Reply:
Removed to avoid confusion.